Makefile

```diff
@@ -6,15 +6,15 @@ CRD_OPTIONS ?= "crd:trivialVersions=true"
 all: manager
 
 # Run tests
-test: generate fmt vet manifests
+test: lint generate fmt vet manifests
```

Comment:
What is your logic for making `lint` a separate `make` target?

Reply:
Contributors are able to run `make lint` before submitting code reviews.

Reply:
Why not make `vet` have `lint` as a target, so that all of these other target dependencies are more consolidated?